src/inet/tests/TestInetLayer.h

```diff
@@ -33,7 +33,8 @@ int TestInetAddress(void);
 int TestInetBuffer(void);
 int TestInetErrorStr(void);
 int TestInetTimer(void);
-
+int TestInetEndPoint(int argc, char * argv[]);
```

Comment:
These don't follow the signature of those above. We don't have any use cases that pass arguments to these at present.

Reply:
Done, thank you